src/vocgui/static/js/corporate_identity.js

```diff
@@ -0,0 +1,26 @@
+/*
+Adapting the footer to display "Tür an Tür Digitalfabrik gGmbH" instead
+of "Jazzmin Version".
+*/
+document
+    .getElementsByTagName("footer")[0]
+    .getElementsByTagName("div")[0]
+    .innerHTML = "<b>Tür an Tür Digitalfabrik gGmbH</b>";
```

Comment:
We did stuff like that on WordPress all the time. These fixes pile up over time and make code really not cool :D I think it works for now, but it would be much cooler to fix this with a pull request in the upstream Jazzmin repository and add a configuration option there.